aci/resource_aci_infrarsdomp.go

```diff
@@ -0,0 +1,151 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/ciscoecosystem/aci-go-client/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAciInfraRsDomP() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAciInfraRsDomPCreate,
+		ReadContext:   resourceAciInfraRsDomPRead,
+		DeleteContext: resourceAciInfraRsDomPDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciInfraRsDomPImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+			"attachable_access_entity_profile_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"annotation": {
```

Comment:
Due to annotation being a optional / modifiable attribute, you will need to add an update function.